codes/classical/properties/Low Density Parity Check (LDPC)

```diff
@@ -0,0 +1,33 @@
+
+code_id: LDPC
+
+name: 'Asymptotically Good LDPC Convolutional Codes Based on Protographs'
```

Comment:
I am not sure why 'Asymptotically Good LDPC Convolutional Codes Based on Protographs' is entered as the name. LDPC convolutional codes is a class of convolutional codes introduced in a paper by Felstrom and Zigangirov around 1999. Codes based on protographs is not a special, well defined class of codes. Asymptotically good may apply to most code families, and should not appear in the description.

Reply:
The paper referenced in the google docs was 'Asymptotically Good LDPC Convolutional Codes Based on Protographs', so I thought I should describe the code introduced there.